### DISCUSSION TITLE
Can I use regular nvim's windows for REPL?

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [Vigemus/iron.nvim#317] [Q&A]
Hey there. Thanks for this plugin!

My question is the following. Can I use regular nvim windows instead of that of iron? Problem with the latter is that it overlays the window instead of actual splitting, so if I have text that was supposed to be wrapped, it isn't, and is left behind REPL.

Here's a screenshot on the border between regular window and REPL. Note where the cursor is - it's on my actual code, which is hidden behind the repl
![image](https://user-images.githubusercontent.com/31178401/218823181-c853967a-1fbf-422d-b536-ed58a8ae5483.png)

**Comment by elliottgreen** [comment]
I have been experiencing a different problem. However, seeing this I realized something. 
For me, the 'repl_open_cmd' is failing to actually pass a split successfully. 
As a result, again thanks to you, I realize that my nvim window is just hidden 'behind' the repl. 

So I don't have an answer for you, but I thank you for your question and how it has helped me!

Thanks :-)

**Reply by WhiteBlackGoose** [reply]
Ohhh, I also got lost initially. It's beacuse the default value for the size is 40, which is ridiculous. You'd have to have a vertical 25"+ monitor so that this size would occupy half of the window lol

**Comment by nwpulverizer** [comment]
I know this is an old post but I am not understanding why anyone would want it to be on top of the normal windows instead of an actual split like you said.
